das_client/lib/app/pages/journey/train_journey/widgets/header/main_container.dart

```diff
@@ -6,31 +7,40 @@ import 'package:das_client/app/widgets/widget_extensions.dart';
 import 'package:design_system_flutter/design_system_flutter.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/flutter_svg.dart';
+import 'package:das_client/model/journey/journey.dart';
 
 class MainContainer extends StatelessWidget {
   const MainContainer({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return SBBGroup(
-      margin: const EdgeInsetsDirectional.fromSTEB(
-        sbbDefaultSpacing * 0.5,
-        0,
-        sbbDefaultSpacing * 0.5,
-        sbbDefaultSpacing,
-      ),
-      padding: const EdgeInsets.all(sbbDefaultSpacing),
-      useShadow: false,
-      child: Column(
-        mainAxisSize: MainAxisSize.min,
-        crossAxisAlignment: CrossAxisAlignment.stretch,
-        children: [
-          _topHeaderRow(),
-          _divider(),
-          _bottomHeaderRow(),
-        ],
-      ),
-    );
+    final bloc = context.trainJourneyCubit;
+
+    return StreamBuilder<Journey?>(
+        stream: bloc.journeyStream,
+        builder: (context, snapshot) {
+          final Journey? journey = snapshot.data;
```

Comment:
hier direkt auf null prüfen, ohne journey macht es wenig sinn den screen to rendern, dann muss es später im _nextStopText nicht mehr machen.